testing/EstNumJellies.py

```diff
@@ -56,9 +64,20 @@ def set_world_pop(self, people):
     def set_frac_ppl_loving_pink(self, frac):
 
         # NE24: Add a test for type here
+        assert type(frac) is float, \
+            "Error: fraction of land set must be a float."
+            
+        # Checking to make sure the fraction isn't zero?
+        try:
+            x = 1/frac
+            except ZeroDivisionError:
```

Comment:
the except needs to be indented at the same level as try. A great way to check is to make your try fail and see what happens.